{"name": "test_add_error[test01]", "status": "passed", "parameters": [{"name": "a", "value": "'æ–‡'"}, {"name": "b", "value": "0.3"}, {"name": "expect", "value": "'TypeError'"}], "start": 1648449816438, "stop": 1648449816438, "uuid": "756d29e3-efd1-41ed-a649-eb6a3f0b9d98", "historyId": "e8365f27c77897acfbd0b0e5d97f78ce", "testCaseId": "a3bfe621045782c9ea9a5de99829cd04", "fullName": "test_pytest.test_cal.TestCal#test_add_error", "labels": [{"name": "parentSuite", "value": "test_pytest"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestCal"}, {"name": "host", "value": "IT-L-YANGYL"}, {"name": "thread", "value": "12580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest.test_cal"}]}